js/Modules/newModule.js

```diff
@@ -1,30 +1,50 @@
 interviewApp.Modules.NewModule = {
+   
     init(id = '') {
+        
+        const currentModule = sessionStorage.getItem('currentModule') || false;
```

Comment:
This block of code looks oddly familiar. It's remarkably similar to that in `logInModule.js`, isn't it? Can we try to abstract it away somehow, so it's nice and DRY?